packages/aws-fargate/package.json

```diff
@@ -68,9 +69,7 @@
     "@instana/metrics-util": "2.33.1",
     "@instana/serverless": "2.33.1"
   },
-  "devDependencies": {
-    "no-code2": "2.0.0"
```

Comment:
FYI: There is no need to keep `node-code2` anymore. We've added it in the past because lerna had a bug with updating the package lock file correctly.

Reply:
I think we also had this as a workaround to make sure there is a node_modules folder in each package. But maybe I remember that wrong, not sure. If it was because of that, the new solution (`node_modules:exist`) is better anyway.